codespell_lib/data/dictionary.txt

```diff
@@ -7880,6 +7890,9 @@ daty->data, date,
 daugher->daughter
 DCHP->DHCP
 dcoker->docker
+dokcer->docker
+dokcre->docker
```

Comment:
Could also add `dcok->dock` and `dokc->dock` ?
And if you feel like it, I guess you could add similar corrections for the `docks`, `docked` and `docking` variants?

Reply:
@lurch done @ 04eec2b4f5c5198bc97e08a230b096a934dd6dcb